src/main/java/frc/robot/commands/ToggleBrakingMode.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+
+public class ToggleBrakingMode extends CommandBase {
+  /** Creates a new ToggleBrakingMode. */
+  private final Drivetrain drivetrain;
+  
+  public ToggleBrakingMode(Drivetrain drivetrain) {
+    this.drivetrain = drivetrain;
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    drivetrain.toggleBrakingMode();
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {}
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
```

Comment:
This command performs a one time action, so it should return true from `isFinished` immediately.